core/stream/fmts/abgr.go

```diff
@@ -17,6 +17,26 @@ package fmts
 import "github.com/google/gapid/core/stream"
 
 var (
+	ABGR_U4_NORM = &stream.Format{
```

Comment:
Please add these formats to the format test package.
core/stream/fmts/fmts_test.go